src/main/java/sysc4806/project24/mini_survey_monkey/models/Question.java

```diff
@@ -0,0 +1,46 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Inheritance(strategy=InheritanceType.JOINED)
+public abstract class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int questionId;
+
+    private String questionTitle;
```

Comment:
**Could we change the variable name `questionTitle` to `question`?**

`questionTitle` isn't quite what we are describing. Questions don't have titles. 

Our variables names should accurately represent how we as humans speak about the domain. For example, `"What is the meaning of life?"` is not a `questionTitle`. We would simply call it a`question`.